### ISSUE TITLE
Kolayana Gate 1.8.0.1 Unlocked

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Witchery#63]
Still is unlocked at the beginning of the game.

**Comment by teagan75** [comment]
That makes no sense. I replaced all the region files in the template. It would only be fixed on a fresh game.

**Comment by satanKami** [comment]
thats weird...when i tested it i had to use the key multipull time to unlock the gate... (ill try again in a new save today)

**Comment by Ladysarajane** [comment]
It was a new world, new files. I had deleted all my files etc, in trying to download the 1.8.0.1. I started the game, grabbed my bag from the tent, Went into Barovia after triggering the location quest for Barovia,  clicked on the gate and it opened. I will try a new world to see. 

Edit..I also forgot to mention, Did you  remove Chicken chunks loader? When world loads, forge keeps asking about the missing Chicken Chunks loader blocks. I reloaded another world and the gate was still unlocked. 
Deleted everything again, started fresh with new files again. Received the Forge Warning missing Chickenchunks loader blocks (I know you took those out, but it is still trying to find them) Started a new world walked into Barovia clicked on the gate and it opened. 

@satanKami  this applies to freshly downloaded files, not updated files on my side.
